csharp-package/brainflow/brainflow/board_shim.cs

```diff
@@ -236,6 +236,29 @@ public static int[] get_eog_channels (int board_id)
             return result;
         }
 
+        /// <summary>
+        /// get row indices of EXG channels for this board
+        /// </summary>
+        /// <param name="board_id"></param>
+        /// <returns>array of row nums</returns>
+        /// <exception cref="BrainFlowException">If this board has no such data exit code is UNSUPPORTED_BOARD_ERROR</exception>
+        public static int[] get_exg_channels (int board_id)
+        {
+            int[] len = new int[1];
+            int[] channels = new int[512];
```

Comment:
What about these? I don't think they are being returned

Reply:
its c#)

Reply:
Yep Nevermind!
It's been a long day 🙃